docs/HowToFAQ/FAQBoogie.md

```diff
@@ -0,0 +1,37 @@
+---
+title: How do I run boogie manually on the Dafny output?
+---
+
+## Question
+
+Running Dafny on my program works, but trying to run Boogie on the bpl output from Dafny fails. What command-line arguments for Boogie am I missing?
+
+## Answer
+
+You can get a Boogie file from Dafny using the option `-print`. For example, the command `dafny -print:a.bpl HelloWorld.dfy` will produce a file `a.bpl`. 
+If the `-print` option lacks an argument (the file name), the somewhat confusing message `Error: No input files were specified` is emitted.
+You can also use `-print:-` to send the boogie file to the standard output.
+
+To run boogie, it is most convenient to install boogie directly. See https://github.com/boogie-org/boogie.
+Dafny invokes Boogie as a library, not as a spawned executable, so there is no Boogie executable in the Dafny installation.
+However, the version of Boogie that Dafny uses corresponds to Boogie 2.15.7 (as of Dafny 3.7.3). 
+
+
+On a simple _Hello World_ program, `boogie a.bpl` is sufficient. Dafny does rely on these default Boogie options
+- `smt.mbqi` is set `false`
+- `model.compact` is set `false`
+- `model.v2` is set `true`
+- `pp.bv_literals` is set `false`
+
+Dafny also sets these non-default Boogie options:
+- `auto_config` is set `false`
+- `type_check` is set `true`
+- `smt.case_split` is set `3`
+- `smt.qi.eager_threshold` is set `100`
+- `smt.delay_units` is set `true`
+- `smt.arith.solver` is set `2`
```

Comment:
None of these are Boogie options though, they are all Z3 options.  I think we can remove them

Reply:
But boogie sets them. If I were trying to run boogie/Z3 manually and reading this, I'd prefer the answer err on the side of pedantic completeness.